geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/ZUnionStoreExecutor.java

```diff
@@ -36,6 +36,11 @@
 
 public class ZUnionStoreExecutor extends AbstractExecutor {
 
+  /**
+   * The position in the command array where the source set keys start.
+   */
+  private static final int SOURCE_KEY_OFFSET = 3;
```

Comment:
This constant is never used.

Reply:
Doh! - Too much ^Z after some experimental code.